### PR TITLE
Show warning when Pod source uses unencrypted HTTP

### PR DESCRIPTION
**Pull request by KrauseFx**
This PR addresses the first half of #7238, related to #7250 

Via https://github.com/CocoaPods/CocoaPods/pull/7249

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,8 +20,24 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
...
+
 ##### Bug Fixes
 
+* Deduplicate output path file names for resources and frameworks  
+  [Eric Amorde](https://github.com/amorde)
+  [#7259](https://github.com/CocoaPods/CocoaPods/issues/7259)
+
+* Allow installation of a pod with its own Swift version on multiple targets  
+  [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
+  [#7261](https://github.com/CocoaPods/CocoaPods/pull/7261)
```
Comment (amorde) [inline CHANGELOG.md]: It seems this changes were unintentional - may need to rebase against master

Comment (orta) [reply CHANGELOG.md]: @amorde can you send me an email saying "Hi" (my emails in on profile, want to invite you to the CocoaPods slack ) 👍

Comment (dnkoutso) [reply CHANGELOG.md]: > It seems this changes were unintentional - may need to rebase against master

yeap

**Comment by KrauseFx** [comment]
Let's try it again https://github.com/CocoaPods/CocoaPods/pull/7293

**Pull request closed without merging**